spec/amp-cache-transform.md

```diff
@@ -78,31 +78,69 @@ content.
 
 For each identifier:
 
- 1. If the identifier contains any parameters, then this identifier cannot be
-    satisfied. The server should attempt to match the next one. (This reserves
-    the parameter space for future additional constraints to be defined.)
- 2. If the identifier is `any`, then the SXG is not intended for a particular
+ 1. If the identifier contains a `v` parameter, then its value represents a set
+    of AMP transform versions. The server should respond with an SXG only if it
+    can produce a one of the versions in that set (see [Version negotation](#version-negotiation)).
+ 2. If the identifier contains any other parameters, then this identifier cannot
+    be satisfied. The server should attempt to match the next one. (This
+    reserves the parameter space for future additional constraints to be
+    defined.)
+ 3. If the identifier is `any`, then the SXG is not intended for a particular
     prefetching intermediary, and therefore its subresource URLs needn't be (but
     may be) rewritten.
- 3. Otherwise, if the identifier is an `id` from the list in
+ 4. Otherwise, if the identifier is an `id` from the list in
     [caches.json](../caches.json), then the SXG should have its subresource URLs
     rewritten. That `id`'s corresponding `cacheDomain` indicates the
     fully-qualified domain name that forms the basis for the URL rewrites.
- 4. Otherwise, the identifier is invalid and cannot be satisfied. The server
+ 5. Otherwise, the identifier is invalid and cannot be satisfied. The server
     should attempt to match the next one.
 
 The server should ensure its copy of `caches.json` is no more than 60 days
 out-of-date with the canonical linked above.
 
+#### Version negotation
+
+This section uses the ABNF rules of
+[RFCF5234](https://tools.ietf.org/html/rfc5234), augmented with the list
+extension defined in [RFC7230 section 7](https://tools.ietf.org/html/rfc7230#section-7),
+the OWS rule from [RFC7230 section 3.2.3](https://tools.ietf.org/html/rfc7230#section-3.2.3),
+and the "sh-" rules in [header-structure-07](https://tools.ietf.org/html/draft-ietf-httpbis-header-structure-07).
+
+The `v` parameter value must be a string. Its value (after parsing as a string)
+must conform to the following ABNF:
+
+```
+v_spec = #v_range
+v_range = sh-integer / sh-integer OWS ".." OWS sh-integer
+```
+
+Each `sh-integer` must be non-negative. If the parameter fails to meet these
+criteria, then the server cannot satisfy the request. Otherwise, the server can
+satisfy the request if, for any `v_range` in the list, any of:
```

Comment:
There are three commas in this sentence :) Maybe move the embedded to the beginning?

`Otherwise, for any v_range in the list, the server can satisfy the request if any of:`

Reply:
Done.